### PR TITLE
Feat - added attempts counter

### PR DESCRIPTION
**Pull request by geea-develop**
Screenshot 2024-07-19 at 11 56 10

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -2,45 +2,64 @@ import "./css/App.css";
...
+                    >
+                        🔄
+                    </button>
+                    <Board
+                        size={getSelectedSize}
+                        chosenImages={getImages}
+                        handleUpdate={handleUpdate}
+                    />
+                    {counter > 0 && (
+                        <div className="eden-text">
```
Comment (YoadBarok) [inline src/App.tsx]: This could be in a separate component 😃

File: src/css/App.css [hunk 2]
```diff
@@ -22,3 +22,7 @@
   background-color: inherit;
   border: none;
 }
+
+.eden-text {
```
Comment (YoadBarok) [inline src/css/App.css:26]: Very good name choice 😎

**Review by YoadBarok — CHANGES_REQUESTED** [review]
Looks good! Let's just move the counter to a component 💪🏻

File: src/App.tsx [hunk 3]
```diff
@@ -2,45 +2,64 @@ import "./css/App.css";
...
+                        onClick={() => window.location.reload()}
+                    >
+                        🔄
+                    </button>
+                    <Board
+                        size={getSelectedSize}
+                        chosenImages={getImages}
+                        handleUpdate={handleUpdate}
+                    />
+                    {counter > 0 && (
```
Comment (YoadBarok) [inline src/App.tsx]: Maybe better to show the counter even when the value is 0 so it will be clear from the beginning that there turns are counted

Comment (geea-develop) [reply src/App.tsx]: Done

File: src/App.tsx (hunk 1 above)
Comment (geea-develop) [reply src/App.tsx]: Done

**Review by YoadBarok — APPROVED** [review]

**Pull request merged**